### PR TITLE
Docs: getting started with netdata cloud onprem

### PR DESCRIPTION
**Pull request by M4itee**
##### Summary

This is the instruction on how to configure helm to be able to pull the helm chart for the Netdata Cloud On-Prem offering.
##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>
#ref netdata/infra#3790

### CODE REVIEW

**Comment by Ancairon** [comment]
Where do we want to present the file in Learn? In which category :thinking:

File: docs/netdata-cloud-onprem/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Getting started with Netdata Cloud On-Prem
+Helm chart for the Netdata Cloud On-Prem installation on Kubernetes is available at ECR registry.
+ECR registry is private, so you need to login first. Credentials are sent by our Marketing team. If you do not have them, please contact our marketing team - marketing@netdata.cloud.
```
Comment (andrewm4894) [inline docs/netdata-cloud-onprem/getting-started.md:3]: i think we should use a better email here marketing@ does not seem right for this. @netdata/product thoughts?

Comment (andrewm4894) [inline docs/netdata-cloud-onprem/getting-started.md:3]:
```suggestion
ECR registry is private, so you need to login first. Credentials are sent by our Product team. If you do not have them, please contact our marketing team - info@netdata.cloud.
```

File: docs/netdata-cloud-onprem/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+```bash
+aws ecr get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin 362923047827.dkr.ecr.us-east-1.amazonaws.com/netdata-cloud-onprem
+```
+
+After this step you should be able to add the repository to your helm or just pull the helm chart:
+```bash
+helm pull oci://362923047827.dkr.ecr.us-east-1.amazonaws.com/netdata-cloud-onprem
+```
+
+Further instructions on how to proceed with the installation are located in the `README.md` file in the chart directory.
```
Comment (andrewm4894) [inline docs/netdata-cloud-onprem/getting-started.md:20]: Any way to link to this for those curious or is this hidden behind some auth layer?

**Comment by andrewm4894** [comment]
I think we should have a new category for "On Prem" to give it as much light and discoverability as possible.

sidebar_label=On Prem
description=Documetation and instructions on how to get started with Netdata Cloud On Prem installs.

**Comment by Ancairon** [comment]
> I think we should have a new category for "On Prem"

Given we will have only one item as of now, should it go as a page right after "Architecture"?

And when we have a second page I can make em into a category.

File: docs/netdata-cloud-onprem/getting-started.md (hunk 1 above)
Comment (M4itee) [reply docs/netdata-cloud-onprem/getting-started.md:3]: I agree

File: docs/netdata-cloud-onprem/getting-started.md (hunk 2 above)
Comment (M4itee) [reply docs/netdata-cloud-onprem/getting-started.md:20]: what exactly? To pull the helm you need to auth first

**Comment by M4itee** [comment]
I'm not sure but it might be good to have it under different section at some point (depends if we will have more docs around the topic)

**Comment by M4itee** [comment]
when we can merge it?

**Review by andrewm4894 — APPROVED** [review]

File: docs/netdata-cloud-onprem/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Getting started with Netdata Cloud On-Prem
+Helm chart for the Netdata Cloud On-Prem installation on Kubernetes is available at ECR registry.
+ECR registry is private, so you need to login first. Credentials are sent by our Product team. If you do not have them, please contact our marketing team - info@netdata.cloud.
```
Comment (andrewm4894) [inline docs/netdata-cloud-onprem/getting-started.md:3]:
```suggestion
ECR registry is private, so you need to login first. Credentials are sent by our Product team. If you do not have them, please contact our team - info@netdata.cloud.
```

**Review by sashwathn — APPROVED** [review]

**Pull request merged**